### PR TITLE
feat: Add reliable Id registry

### PR DESCRIPTION
**Pull request by bee344**
Adds a dynamic registry for the Parachains' Ids instead of using a hardcoded one. It tries out each endpoint and if not responsive, moves onto the next one. Also added some cli logs to confirm it is working, since it takes quite a while to fetch the data.
Closes: #3

### CODE REVIEW

File: src/createRegistry.ts [hunk 1]
```diff
@@ -85,54 +98,51 @@ const fetchChainInfo = async (
...
-	);
-
-	const api = await ApiPromise.create({
-		provider: new WsProvider(wsUrls),
-		noInitWarn: true,
-	});
-
-	await api.isReady;
+	const api = await getApi(endpointOpts, isRelay);
+	console.log('Api ', api?.isConnected);
```
Comment (TarikGul) [inline src/createRegistry.ts:102]: If we are going to keep this console here, lets have it be a little more descriptive such as `Api connected: `

File: src/createRegistry.ts [hunk 2]
```diff
@@ -270,6 +314,185 @@ const fetchSystemParachainForeignAssetInfo = async (
...
+				await wsProvider.disconnect();
+				retry = 0;
+			}
+		}
+	}
+
+	if (enpdointArray.length === 0) {
+		return;
+	} else {
+		console.log('array', enpdointArray);
```
Comment (TarikGul) [inline src/createRegistry.ts:490]: Same with this console, if we want it present lets make it a little more descriptive.

File: .gitignore [hunk 3]
```diff
@@ -10,4 +10,4 @@
 !.yarn/plugins
 
 # Tsc Build
-**/build
+**/build
```
Comment (TarikGul) [inline .gitignore:13]: New line.

**Review by TarikGul — APPROVED** [review]
Overall, fantastic PR, and I think this is going to be incredibly more reliable for connecting to parachains! Great job

File: .gitignore (hunk 3 above)
Comment (bee344) [reply .gitignore:13]: Could you elaborate on this comment?

File: src/createRegistry.ts (hunk 2 above)
Comment (bee344) [reply src/createRegistry.ts:490]: Removed it, it was a leftover from development debugging.

File: src/createRegistry.ts (hunk 1 above)
Comment (bee344) [reply src/createRegistry.ts:102]: Done, added in the next commit.

**Review by marshacb — APPROVED** [review]
great PR! no additional nits.

**Pull request merged**